src/Form/CompanyType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Address;
+use App\Entity\Company;
+use App\Entity\LegalForm;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CompanyType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('denomination')
```

Comment:
@DaniOmer Validations are missing

Reply:
It's normal. 
La PR c'est pour la validation au niveau de customer. J'ai créer le crud company juste pour avoir les routes de redirection